TA-access.md

```diff
@@ -0,0 +1,68 @@
+# Getting Access to Openshift Console
+
+## Login
+
+Open this link in a browser:
+
+- <https://console.apps.shift.nerc.mghpcc.org/k8s/cluster/projects/rhods-notebooks>
+
+Select mss-keycloak login option:
+
+![](images/openshift-login.png)
+
+You will be redirected to a site managed by CILogon where you will select your institutional or commercial identity provider as shown below (Choose Boston University if you are using your BU ID):
+
+![](images/select-provider.png)
+
+Once selected, you will be redirected to your institutional or commercial identity provider where you will login as shown here:
+
+![](images/bulogin.png)
+
+## Openshift Web Interface
+
+After successful logon, your browser will direct you to NERC Openshift Console rhods-notebooks namespace. Here, you have the ability to view and manage workloads, as well as search for and list notebooks. Please note that the workloads and notebooks displayed include from both cs210 and ee440 classes. It’s important to be aware of this broader scope while managing resources in the namespace.
```

Comment:
Should we reference Lance's class here as well?

Reply:
Done!